codegen/src/test/java/software/amazon/awssdk/codegen/poet/ClientTestModels.java

```diff
@@ -229,6 +229,21 @@ public static IntermediateModel serviceMiniS3() {
         return new IntermediateModelBuilder(models).build();
     }
 
+    public static IntermediateModel serviceMixedAuth() {
```

Comment:
it's confusing to see serviceMixedAuth, but then customresponsemetadata as the c2j file.

Reply:
Also, what new case are you covering in this model, that isn't covered in say fine-grained-auth? Could it be combined into an existing model?

Reply:
This case is covering the added code that converts from the legacy `authType` with a single value to the new `auth` with multiple values when the later is not specified.

Reply:
> it's confusing to see serviceMixedAuth, but then customresponsemetadata as the c2j file.
I will rename to a more meaningful one.

Reply:
hmm, not fine-grained-auth, but isn't this case defined in other existing models. I think c2j/json/service-2.json or c2j/restjson/service-2.json have the same use case.

Reply:
`rest-json` has one operation with bearer but we are not using that model. I will keep this one separated but simplify the model as suggested somewhere else.